### PR TITLE
Us113777/error page

### PR DESCRIPTION
**Pull request by louiscoste**
image
image

### CODE REVIEW

**Comment by quinnbudan** [comment]
Nit: The "Uh oh!" looks bigger than the mockup, I know we had this conversation with the success page but I don't remember the outcome of that conversation

File: src/components/widget/d2l-teacher-course-creation-error.js [hunk 1]
```diff
@@ -28,13 +58,40 @@ class TeacherCourseCreationError extends BaseMixin(LitElement) {
...
 	connectedCallback() {
 		super.connectedCallback();
+
+		if (this.pageData && this.pageData.ErrorMessage) {
+			this.errorMessage = this.pageData.ErrorMessage;
+		}
+	}
+
+	_backPressed() {
+		this.changePage(PAGES.INPUT_PAGE);
```
Comment (quinnbudan) [inline src/components/widget/d2l-teacher-course-creation-error.js:68]: If it goes back to the input page, then I think it should pass back the page data object

**Comment by louiscoste** [comment]
We kept the this size. But we should do a full UI review with Chris at the end once we have all the pages because I think some changes will spam across multiple pages

File: locales/en.js [hunk 2]
```diff
@@ -9,6 +9,10 @@ export const val = {
 	'columnPrefix': 'Prefix',
 	'columnSuffix': 'Suffix',
 	'columnRole': 'Role',
+	'errorBackButtonDescription': 'Back to creating a course',
+	'errorBackButtonText': 'Back',
```
Comment (TomMichielsD2L) [inline locales/en.js:13]: Lets use 'actionBack' instead of making another langTerm

File: test/d2l-custom-teacher-course-creation-error.test.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+		});
+	});
+
+	describe('constructor', () => {
+		it('should construct', () => {
+			runConstructor('d2l-tcc-error');
+		});
+	});
+
+	describe('button actions', () => {
```
Comment (TomMichielsD2L) [inline test/d2l-custom-teacher-course-creation-error.test.js:21]: Perhaps we can also add a test to check the errorMessage binding?

**Pull request merged**